staging/src/k8s.io/client-go/tools/cache/reflector.go

```diff
@@ -68,6 +68,23 @@ type Reflector struct {
 	// WatchListPageSize is the requested chunk size of initial and resync watch lists.
 	// Defaults to pager.PageSize.
 	WatchListPageSize int64
+
+	// MinimumResourceVersion sets the earliest allowed change a reflector's initial list and watch may start from.
```

Comment:
I'd suggest avoiding language that implies all values must be numbers. Something like `OldestAllowedResourceVersion` might be better (though I don't love the name).